src/Css/audienceBrowser.css

```diff
@@ -0,0 +1,108 @@
+* {
+    font-family: Faktum, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+    text-decoration: none;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.listContainer {
+    margin: 15px 30px;
+    max-width: 500px;
+    min-height: 600px;
+    background-color: lightgray;
+    border-radius: 5px;
+    border: 1px solid grey;
+    padding: 10px;
+    list-style-type: none;
+}
+
+.list {
+    list-style-type: none;
+}
+
+.folder {
+    width: 100%;
+    min-height: 40px;
+    background-color: mediumvioletred;
+    color: white;
+    margin: 10px auto 0 auto;
+    padding: 5px 5px;
+    border-radius: 5px;
+    line-height: 10px;
+    text-align: left;
+    border: 2px solid mediumvioletred;
+
+}
+
+.folder:hover {
+    background-color: rgb(226, 116, 186);
+
+}
+
+.audience {
+    width: 100%;
+    min-height: 40px;
+    background-color: rgb(255, 216, 241);
+    color: rgb(59, 59, 59);
+    margin: 10px auto 0 auto;
+    padding: 5px 5px;
+    border-radius: 5px;
+    line-height: 10px;
+    text-align: left;
+    border: 2px solid mediumvioletred;
+}
+
+.audience:hover {
+    background-color: rgb(255, 255, 255);
+
+}
+
+.audience:focus {
+    background-color: rgb(255, 255, 255);
+    color: mediumvioletred;
+}
+
+.h1 {
+    margin: 20px auto 10px 30px;
+    font-size: 24px;
+}
+
+.backButton {
+    display: inline-block;
+    width: 20%;
+    background-color: mediumvioletred;
+    color: white;
+    padding: 5px 5px;
+    border-radius: 5px;
+    line-height: 10px;
+    text-align: center;
+    border: 2px solid mediumvioletred;
+}
+
+.backButton:hover {
+    background-color: rgb(226, 116, 186);
+
+}
+
+.breadcrumbsArea {
+    display: inline-block;
+    width: 80%;
+}
+
+.breadcrumbsText {
+    display: inline;
+    line-height: 10px;
+}
+
+.breadcrumbs {
+    display: inline;
+    line-height: 10px;
+}
+
+#head {
+    width: 100%;
+    min-height: 40px;
+    border-bottom: 1px solid grey;
+    padding: 5px 5px 10px 5px;
+}
```

Comment:
It's good practice to do not use HTML IDs for styling, we have our internal guideline for styling https://github.com/GlobalWebIndex/weak-css